components/Frame/Header.js

```diff
@@ -203,13 +204,17 @@ const isPositionStickySupported = () => {
 // - iOS 11.4: header is transparent and only appears after triggering a render by scrolling down enough
 const forceRefRedraw = ref => {
   if (ref) {
-    setTimeout(() => {
+    const redraw = () => {
       const display = ref.style.display
       ref.style.display = 'none'
       /* eslint-disable-next-line no-unused-expressions */
       ref.offsetHeight
       ref.style.display = display
-    }, 300)
+    }
+    setTimeout(redraw, 33)
```

Comment:
Comments: Would leave some comments here. Questions expected to answer: Why 33ms? Why multiple times with some backoff logic?

Logic: Maybe even wrap that into an array that get's mapped.